server/types/types.go

```diff
@@ -118,6 +118,9 @@ func (p PolicyV1) Equal(other PolicyV1) bool {
 	return p.ID == other.ID && p.Raw == other.Raw && p.AST.Equal(other.AST)
 }
 
+// ProvenanceV1 models a collection of build/version information.
+type ProvenanceV1 map[string]interface{}
```

Comment:
Nit: This type is unused other than below. In the server code you could make use of this type. One option would be to define a struct for the provenance information like so:

```string
type ProvenanceV1 struct {
  Version string `json:"version"
  Revision string `json:"revision,omitempty"`
}
```

And then instantiate/return that above from `getProvenance()`. The response writing _should_ just do the right thing.

I'd prefer to either go with the struct approach OR just use `map[string]interface{}` everywhere to avoid confusion.

Reply:
+1 for a struct, this should probably be a well defined return value.

Reply:
understood, will update the patch-set.